Test/git-issues/git-issue-532.dfy.expect

```diff
@@ -0,0 +1,27 @@
+git-issue-532.dfy(25,19): Warning: /!\ No terms found to trigger on.
```

Comment:
These no-trigger warnings can be suppressed as I show in the additional test cases.

Reply:
Fixed as described